### PR TITLE
Chore: Support for Screen Reader on PC and Xbox

### PR DESCRIPTION
**Pull request by DeekyJay**

### CODE REVIEW

**Review by ProbablePrime — APPROVED** [review]

File: src/alchemy/preact/Cooldown.tsx [hunk 1]
```diff
@@ -54,13 +54,17 @@ export class CoolDown extends Component<
           cActive: this.state.ttl >= 0,
           progress: !!this.props.progress,
         })}
+        role="status"
```
Comment (connor4312) [inline src/alchemy/preact/Cooldown.tsx]: I think the [progressbar role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_progressbar_role) would actually be more appropriate here

```suggestion
        role="progressbar"
        aria-valuenow={this.state.progress}
        aria-valuemax={0}
        aria-valuemin={1}
        aria-valuetext={`Cooling down for ${this.state.ttl} seconds`} 
```

File: src/alchemy/preact/SparkPill.tsx [hunk 2]
```diff
@@ -30,8 +30,13 @@ export class SparkPill extends Component<
           unaffordable: this.props.cost > this.props.available,
         })}
         style={styles}
+        role="presentation"
```
Comment (connor4312) [inline src/alchemy/preact/SparkPill.tsx]: role=presentation hides it from screen readers, I think you want to omit that

```suggestion
```

File: src/prefabs/button/button.tsx [hunk 3]
```diff
@@ -37,6 +37,8 @@ export class ProgressBar extends Component<{ value: number }, {}> {
           style={css({
             transform: `translateX(${width * 100}%)`,
           })}
+          role="status"
```
Comment (connor4312) [inline src/prefabs/button/button.tsx]: progressbar here perhaps

**Comment by AechDub** [comment]
What @connor4312 said, otherwise gtg

**Review by AechDub — APPROVED** [review]

File: src/alchemy/preact/SparkPill.tsx (hunk 2 above)
Comment (DeekyJay) [reply src/alchemy/preact/SparkPill.tsx]: oh interesting, Chrome was reading it, but Edge was not. I'll make this change and see if I can remove some of my Edge specific logic.

**Pull request merged**